src/tests/tribol_mortar_sparse_weights.cpp

```diff
@@ -75,12 +67,10 @@ void computeGapsFromSparseWts( tribol::CouplingScheme const * cs, real * gaps )
    SLIC_ERROR_IF(I==nullptr, "Mortar wts test, I is null.");
 
    // get mortar node id offset to distinguish mortar from nonmortar column contributions
-   if (mortarMesh.m_sortedSurfaceNodeIds == nullptr)
-   {
-      SLIC_DEBUG("computeGapsFromSparseWts(): sorting unique mortar surface node ids.");
-      mortarMesh.sortSurfaceNodeIds();
-   }
-   int nodeOffset = mortarMesh.m_sortedSurfaceNodeIds[ mortarMesh.m_numSurfaceNodes-1 ] + 1;
+   // sorts unique surface node ids from connectivity
+   auto mortar_id = mortarMesh.meshId();
+   auto sorted_surface_node_ids = tribol::MeshManager::getInstance().at(mortar_id).sortSurfaceNodeIds();
```

Comment:
Likely the weights read in are numbered by DOFs used instead of total DOFs?  Is there a difference in how the test works with the new changes?  It looks like it's just been updated to use the new code.